app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss

```diff
@@ -0,0 +1,95 @@
+.govuk-previous-and-next-navigation {
+  display: block;
+
+  @include media-down(mobile) {
+    margin: 2em 0 0 0;
+  }
+
+  ul {
+    margin: 0;
+    padding: 0;
+  }
+
+  li {
+    @include core-16($line-height: (20 / 16));
+    float: left;
+    list-style: none;
+    text-align: right;
+    margin: 0;
+    padding: 0;
+    width: 49%;
+
+    a {
+      display: block;
+      color: $link-colour;
+      text-decoration: none;
+
+      @include ie-lte(7) {
+        height: 4.5em;
+      }
+
+      &:hover,
+      &:active {
+        background-color: $canvas-colour;
+      }
+
+      .pagination-label {
+        @include core-27($line-height: (33.75 / 27));
+        margin-bottom: 0.1em;
+        display: block;
+      }
+
+    }
+
+    &.next-element {
+      float: right;
+      text-align: right;
+    }
+
+    &.next-element a:before {
+      background: transparent image-url("govuk-component/arrow-sprite.png") no-repeat -102px -11px;
+      margin: -4px -32px 0 0;
+      display: block;
+      float: right;
+      width: 30px;
+      height: 38px;
+      content: " ";
+    }
+
+    &.previous-element a:before {
+      background: transparent image-url("govuk-component/arrow-sprite.png") no-repeat -20px -11px;
+      margin: -4px 0 0 -32px;
+      display: block;
+      float: left;
+      width: 30px;
+      height: 38px;
+      content: " ";
+      }
+
+    &.previous-element {
+      float: left;
+      text-align: left;
+    }
+
+    &.previous-element a {
+      padding: 0.75em 0 0.75em 3em;
```

Comment:
Should spacing be in pixels? https://github.com/alphagov/styleguides/blob/master/css.md#dimensions